src/components/Button/Button.test.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+import React from 'react';
+import { Button } from './Button';
+
+describe('<Button />', () => {
+
+  it('should render button with text', () => {
```

Comment:
@ej9x I think the tests for check render should use snapshots, it is more easy for writing, supporting and reading